### DISCUSSION TITLE
How to: Only allow whitelisted Email-Adresses to fill out a form (without exposing the Email-Adresses on the frontend)?

### DISCUSSION THREAD

**Discussion by NilsDannemann** [nuxt/nuxt#17505] [Questions]
Hey everyone!

I have a simple signup form in an nuxt3 app. 
But only people who are part of our newsletter email-list should be allowed to sign up.

How do I only allow whitelisted Email-Adresses be allowed to fill out this form without exposing the full email list on the frontend?

I'm not sure how to approach this.

Kind Regards,
Nils

**Comment by danielroe** [comment]
You should likely implement this with a [Nitro server route](https://v3.nuxtjs.org/guide/features/server-routes#server-routes).
